tools/e2e-commons/env/prerequisites.js

```diff
@@ -90,6 +104,10 @@ async function buildPrerequisites( state, page ) {
 	}
 }
 
+/**
+ * Ensure connected state.
+ * @param {boolean} requiredConnected - Whether the site should be connected.
+ */
 export async function ensureConnectedState( requiredConnected = undefined ) {
```

Comment:
```suggestion
export async function ensureConnectedState( requiredConnected = null ) {
```

Reply:
I could see `false`, but why `null`?

Reply:
Well, my main aversion was toward `undefined`. Going with `null` to me means "no default" (allowing one to handle that specific case in code), and with `false` it means an opinionated default. :^)

Feel free to use either one or just ignore.

```suggestion
export async function ensureConnectedState( requiredConnected = false ) {
```